packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -405,7 +405,7 @@ function* sendTipAsync() {
         senderHandle: sender.handle,
         recipientHandle: receiver.handle,
         amount,
-        error: e.message,
+        error: 'transactionMessage' in e ? e.transactionMessage : e.message,
```

Comment:
what's this? will we be required to use this every time we want to parse an error on the client?

Reply:
it depends. The reason I do this is because the actual error message includes the signature, which ruins bucketing in amplitude but is useful in sentry